integration_tests/tests/consistency/consistency_qualtrics__contact.sql

```diff
@@ -0,0 +1,49 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with prod as (
+    select
+        {{ dbt_utils.star(from=ref('qualtrics__contact')) }}
+    from {{ target.schema }}_qualtrics_prod.qualtrics__contact
+),
+
+dev as (
+    select
+        {{ dbt_utils.star(from=ref('qualtrics__contact')) }}
+    from {{ target.schema }}_qualtrics_dev.qualtrics__contact
```

Comment:
When running this validation test myself, I ran into a test failure.

When looking into why this failed I found that it was due to the `avg_survey_duration_in_seconds` field having a delta of about 0.00000000000001... Can we update this test to make sure we round to the nearest second for these metrics so this doesn't have false failures in the future.

Reply:
updated to round `avg_survey_duration_in_seconds`